packages/relay/src/RegistrationManager.ts

```diff
@@ -369,7 +369,7 @@ export class RegistrationManager {
   async addRelayWorker (currentBlockNumber: number, currentBlockTimestamp: number): Promise<PrefixedHexString> {
     // register on chain
     const addRelayWorkerMethod = await this.contractInteractor.getAddRelayWorkersMethod([this.workerAddress])
-    const gasLimit = await this.transactionManager.attemptEstimateGas('AddRelayWorkers', addRelayWorkerMethod, this.managerAddress)
+    const gasLimit = await this.transactionManager.estimateGas('AddRelayWorkers', addRelayWorkerMethod, this.managerAddress)
```

Comment:
its ok, but redundant .
`sendTransaction` itself does `estimateGas()` in case it isn't given a `gasLimit`.
So what is the reason for explicitly estimating gas before calling it?

Reply:
Right. Fiexd.